nuxeo-features/nuxeo-admin-center/nuxeo-admin-center-core/src/main/java/org/nuxeo/ecm/admin/permissions/PermissionsPurgeWork.java

```diff
@@ -98,6 +102,9 @@ public void work() {
         }
 
         List<String> usernames = (List<String>) searchDocument.getPropertyValue("rs:ace_username");
+        log.info("Purging permissions on {} documents for usernames: {} and ancestorIds: {}", docIds::size,
```

Comment:
Is this left at INFO while debugging the issue? Otherwise I think DEBUG would be better.

Reply:
We thought INFO was OK:
- This code is not heavily called
- Most of the production servers should log WARN and not INFO
- Yes, It could help while debugging the issue as we log INFO in the unit tests

I'm not against logging it at DEBUG level, but that means change also the logging threshold in the unit tests ... and I'm afraid we will forget to put it back to INFO :)